src/main/webapp/about.jsp

```diff
@@ -66,5 +66,25 @@
       </p>
     </div>
   </div>
+  
+<div id="container">
+	<div
+       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+       
+       <h1>Team 21 Info</h1>
+       
+       <ul>
+         <li><strong>(This is just a git operation test XD )</strong>
+         </li>
+         <li><strong>Sherry Yang</strong> I am a sophomore studying at University of Washington,
```

Comment:
Can you deploy your change locally and attach a screenshot of the new page for UI changes?  Thanks :)

Reply:
![image](https://user-images.githubusercontent.com/10622320/37183702-3f040b68-22ec-11e8-8e79-daf757a54cd2.png)